src/Social/Models/UserMessages.php

```diff
@@ -27,11 +28,13 @@ public function initialize()
      */
     public function getUserFeeds(UserInterface $user): Simple
     {
-        $userFeeds = new Simple(
+       $appData = Di::getDefault()->get('app');
+
+       $userFeeds = new Simple(
             null,
             new Messages(),
             $this->getReadConnection()->query(
-                "SELECT * from messages where id in (SELECT messages_id from user_messages where users_id = {$user->getId()} and is_deleted = 0)"
+            "SELECT * from messages where id in (SELECT messages_id from user_messages where users_id = {$user->getId()} and is_deleted = 0) and apps_id = {$appData->getId()}"
```

Comment:
@arfenis nop this query we should check with @SparoHawk to optimize and we also need to paginate it